assets/css/editor.scss

```diff
@@ -39,3 +39,10 @@
 		}
 	}
 }
+
+// Ensure textarea bg color is transparent for block titles.
+// Some themes (e.g. Twenty Twenty) set bg white for textarea, which breaks editor theme styles.
```

Comment:
>Some themes (e.g. Twenty Twenty) set bg white for textarea, which breaks editor theme styles.

The problem was not from Twenty Twenty, the problem was from Gutenberg defining a white background for text inputs and Twenty Twenty registering a background color that is not white to the editor, I think I should also fill this issue in Gutenberg later.

Reply:
I was wondering about that, if this is an issue we need to log upstream - thanks for explaining :)